liferay-content-management/liferay-portal
# LPD-31851 Warn the user about UndoRedo History reset when changing default language

LPD-31851 Warn the user about UndoRedo History reset when changing default language

# What is this trying to solve?

We want to add an alert to warn the user when he/she wants to change the defual language.

[Link to ticket](https://liferay.atlassian.net/browse/LPD-31851)

# How am I fixing it?

- I have added the alert in the component and adapted the test.

# How can I test it?

- Enable FF 'LPD-11228': true,
- Go to System Settings > Web Content > Administration > Enable Changeable default language
- Go to Content & Data > Create a Web Content > See the alert appears
![warning](https://github.com/user-attachments/assets/5c218d0c-d39a-412e-9f18-c8c5562a98e8)
